pkg/api/graph/graphview/veneering_test.go

```diff
@@ -316,7 +320,7 @@ func TestGraph(t *testing.T) {
 		}
 	}
 
-	if dc != 2 || bc != 3 || ist != 3 || other != 12 {
+	if dc != 2 || bc != 3 || ist != 3 || other != 15 {
```

Comment:
Git blame shows you changed it, therefore you inherit code guilt

On Jul 7, 2015, at 3:41 PM, David Eads <notifications@github.com> wrote:

In pkg/api/graph/graphview/veneering_test.go
<https://github.com/openshift/origin/pull/3621#discussion_r34081343>:

> @@ -316,7 +320,7 @@ func TestGraph(t *testing.T) {
>  		}
>  	}
>
> -	if dc != 2 || bc != 3 || ist != 3 || other != 12 {
> +	if dc != 2 || bc != 3 || ist != 3 || other != 15 {

 I can see why 12 wasn't working well. 15 is definitely better

I blame @smarterclayton <https://github.com/smarterclayton>. 15 is "the
rest" of the nodes.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3621/files#r34081343>.